internal/artifacts/module_arch.go

```diff
@@ -2,10 +2,10 @@ package artifacts
 
 import (
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
This should also be below this import group

Reply:
fixed